src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrBackflower.cs

```diff
@@ -545,4 +546,8 @@ await _repositoryCloneManager.PrepareCloneAsync(
 
     protected override NativePath GetEngCommonPath(NativePath sourceRepo) => sourceRepo / VmrInfo.SourceDirName / "arcade" / Constants.CommonScriptFilesPath;
     protected override bool TargetRepoIsVmr() => false;
+    // During backflow, we're flowing a specific VMR commit that the build was built from, so we should just check it out
+    protected virtual bool ShouldResetVmr { get; } = false;
+    // When flowing local repos, we should never reset branches to the remote ones, we might lose some changes devs wanted
+    protected virtual bool ShouldResetClones { get; } = false;
```

Comment:
This one is not used in this class?

Reply:
I thought it might be a good idea to have them everywhere even if they're not used, removed, no need for it, just dead code